src/components/core/TextArea/TextArea.tsx

```diff
@@ -27,6 +27,7 @@ export const TextArea = forwardRef<HTMLTextAreaElement, TextAreaProps>(
         placeholder={placeholder}
         defaultValue={defaultValue}
         onChange={onChange}
+        onKeyUp={(e) => e.stopPropagation()} // To prevent keyboard navigation from triggering while in textarea
```

Comment:
@Robinnnnn Now including this in `<TextArea>` and we no longer need any check for focus in `useKeyDown`

Reply:
niceee that's sick!